src/Scanner.java

```diff
@@ -205,6 +207,16 @@
   }
 
   /**
+  * Specifies if the scan will be in reverse or not
+  * @param to_reverse Indication of scan direction. If this is not
+  * invoked, scanning will default to not being reversed
+  */
+  public void setReversed(boolean to_reverse){
```

Comment:
Yes, that would be cleaner. I think I will change it to that way. I just confirmed in the code that once you set a direction for a scanner, you cannot change it without creating a new scan.